STATUS: INLINE SUCCESSFUL
MODIFIED: pitt.search.semanticvectors.vectors.ComplexVectorTest.testComplexOrthogonalize
MODIFIED FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/vectors/ComplexVectorTest.java
CLEANER: pitt.search.semanticvectors.vectors.ComplexVectorTest.testBindFromZero
CLEANER FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/vectors/ComplexVectorTest.java
POLLUTER: pitt.search.semanticvectors.CompareTermsTest.testCompareTermsComplexFlatOrthographic
POLLUTER FILE: /home/lichengpeng/test/automated_script/Lucky-Dhakad/semanticvectors/src/test/java/pitt/search/semanticvectors/CompareTermsTest.java
ORIGINAL CLEANER SIZE: 9
NEW CLEANER SIZE: 2
ELAPSED TIME: 185.89599990844727
==========================
@@ -224,0 +224,4 @@
+ {
+     Random random = new Random(0);
+     ComplexVector cv2 = (ComplexVector) VectorFactory.generateRandomVector(VectorType.COMPLEX, 5, 2, random);
+ }
